_extras/guide.md

```diff
@@ -38,3 +38,10 @@ ____
 * If "https" doesn't work to fetch CrossRef during Advanced OpenRefine Functions, they can try "http"
 
 * If they need to diagnose failure to fetch the content from the URL they can check the "Store error" option in the "Add column by fetching URLs" dialogue and try looking at the common problems listed in the [documentation](https://docs.openrefine.org/manual/columnediting#common-errors)
+
+## Poweful transformations
+
+* In the titlecase exercise, highlight the fact that
+  each transformation can have unintended side effects,
+  and advise that running one cleanup operation too little
```

Comment:
Minor grammar: this should be 'too few' rather than 'too little'

Reply:
😅 Thanks! Amended offline.